sunpy/database/database.py

```diff
@@ -627,6 +627,11 @@ def query(self, *query, **kwargs):
 
         return sorted(db_entries, key=operator.attrgetter(sortby))
 
+    @deprecated('0.8', alternative='database.search')
+    def query(self, *query, **kwargs):
+        __doc__ = self.search.__doc__
```

Comment:
I was lazy and didn't use the sphinx doc link i used for the mapbase changes.

But yes, I have not actually checked it but it exists in the database code. I just built the docs so I'll check.